src/App.js

```diff
@@ -15,25 +41,29 @@ function App() {
 
             {/* TODO STEP 3 - We need to change the hardcoded values in these divs to accept dynamic values from our state. */}
 
-            <div className="home__score">32</div>
+            <div className="home__score"><span>{LionsScore}</span></div>
           </div>
           <div className="timer">00:03</div>
```

Comment:
Miley this looks good. And I think you done a good job. Check into the moment.js to get the timmer working. Heres a little something to read though. https://upmostly.com/tutorials/build-a-react-timer-component-using-hooks